pkg/proxy/service.go

```diff
@@ -59,11 +80,165 @@ func (s *implLoadTestServiceServer) Get(ctx context.Context, in *grpcProxyV2.Get
 	}, nil
 }
 
+// Create creates new load test
+func (s *implLoadTestServiceServer) Create(ctx context.Context, in *grpcProxyV2.CreateRequest) (*grpcProxyV2.CreateResponse, error) {
```

Comment:
Would be better to abstract all the logic inside this function and re-use on both v1/v2 endpoints?
Because now we have two completely different endpoints to maintain.

Reply:
Same thing for the other endpoints

Reply:
I decsribed it in the PR description - it was my first intention, but the whole logic is just simple calls to something and then returning backend-agnostic result (different http errors) in case of the problem. Refactoring this may be separate task.